explorer/src/components/account/NFTokenAccountSection.tsx

```diff
@@ -0,0 +1,267 @@
+    </div>
+  );
+};
+
+export const NftokenImage = ({
+  url,
+  size,
+}: {
+  url: string | undefined;
+  size: number;
+}) => {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [showError, setShowError] = useState<boolean>(false);
+  const [timeout, setTimeout] = useState<NodeJS.Timeout | undefined>(undefined);
+
+  useEffect(() => {
+    // Set the timeout if we don't have a valid uri
+    if (!url && !timeout) {
+      setTimeout(setInterval(() => setShowError(true), MAX_TIME_LOADING_IMAGE));
+    }
+
+    // We have a uri - clear the timeout
+    if (url && timeout) {
+      clearInterval(timeout);
+    }
+
+    return () => {
+      if (timeout) {
+        clearInterval(timeout);
+      }
+    };
+  }, [url, setShowError, timeout, setTimeout]);
+
+  const { cachedBlob } = useCachedImage(url || "");
+
+  return (
+    <>
+      {showError ? (
+        <div
+          style={{ width: size, height: size, backgroundColor: "lightgrey" }}
+        />
+      ) : (
+        <>
+          {isLoading && (
+            <div
+              style={{
+                width: size,
+                height: size,
+                backgroundColor: "lightgrey",
+              }}
+            />
+          )}
+          <div className={`${isLoading ? "d-none" : "d-block"}`}>
+            <img
+              className={`rounded mx-auto ${isLoading ? "d-none" : "d-block"}`}
+              src={cachedBlob}
+              alt={"nft"}
+              style={{
+                width: size,
+                height: size,
+              }}
+              onLoad={() => {
+                setIsLoading(false);
+              }}
+              onError={() => {
+                setShowError(true);
+              }}
+            />
+          </div>
+        </>
+      )}
+    </>
+  );
+};
+
+const CollectionCard = ({
+  collection,
+}: {
+  collection: NftokenTypes.CollectionAccount;
+}) => {
+  const fetchInfo = useFetchAccountInfo();
+  const refresh = () => fetchInfo(new PublicKey(collection.address));
+
+  return (
+    <div className="card">
+      <div className="card-header">
+        <h3 className="card-header-title mb-0 d-flex align-items-center">
+          Overview
+        </h3>
+        <button className="btn btn-white btn-sm" onClick={refresh}>
+          <span className="fe fe-refresh-cw me-2"></span>
+          Refresh
+        </button>
+      </div>
+
+      <TableCardBody>
+        <tr>
+          <td>Address</td>
+          <td className="text-lg-end">
+            <Address
+              pubkey={new PublicKey(collection.address)}
+              alignRight
+              raw
+            />
+          </td>
+        </tr>
+        <tr>
+          <td>Authority</td>
+          <td className="text-lg-end">
+            <Address
+              pubkey={new PublicKey(collection.authority)}
+              alignRight
+              link
+            />
+          </td>
+        </tr>
+        <tr>
+          <td>Number NFTs</td>
```

Comment:
```suggestion
          <td>Number of NFTs</td>
```
alternatively, "Size" would work here too